x-pack/legacy/plugins/siem/public/pages/case/components/case_view/index.tsx

```diff
@@ -0,0 +1,311 @@
+import { UserActionTree } from '../user_action_tree';
+import { UserList } from '../user_list';
+import { useUpdateCase } from '../../../../containers/case/use_update_case';
+import { WrapperPage } from '../../../../components/wrapper_page';
+
+interface Props {
+  caseId: string;
+}
+
+const MyDescriptionList = styled(EuiDescriptionList)`
+  ${({ theme }) => css`
+    & {
+      padding-right: ${theme.eui.euiSizeL};
+      border-right: ${theme.eui.euiBorderThin};
+    }
+  `}
+`;
+
+const MyWrapper = styled(WrapperPage)`
+  padding-bottom: 0;
+  // ${({ theme }) => css`
+  //   @media only screen and (min-width: ${theme.eui.euiBreakpoints.l}) {
+  //     margin: 0 auto;
+  //     width: 85%;
+  //   }
+  // `}
+`;
+const BackgroundWrapper = styled.div`
+  ${({ theme }) => css`
+    background-color: ${theme.eui.euiColorEmptyShade};
+    border-top: ${theme.eui.euiBorderThin};
+    height: 100%;
+  `}
+`;
+
+interface CasesProps {
+  caseId: string;
+  initialData: Case;
+  isLoading: boolean;
+}
+
+export const Cases = React.memo<CasesProps>(({ caseId, initialData, isLoading }) => {
+  const [{ data }, dispatchUpdateCaseProperty] = useUpdateCase(caseId, initialData);
+  const [isEditDescription, setIsEditDescription] = useState(false);
+  const [isEditTitle, setIsEditTitle] = useState(false);
+  const [isEditTags, setIsEditTags] = useState(false);
+  const [isCaseOpen, setIsCaseOpen] = useState(data.state === 'open');
+  const [description, setDescription] = useState(data.description);
+  const [title, setTitle] = useState(data.title);
+  const [tags, setTags] = useState(data.tags);
+
+  const onUpdateField = useCallback(
+    async (updateKey: keyof Case, updateValue: string | string[]) => {
+      switch (updateKey) {
+        case 'title':
+          if (updateValue.length > 0) {
+            dispatchUpdateCaseProperty({
+              updateKey: 'title',
+              updateValue,
+            });
+            setIsEditTitle(false);
+          }
+          break;
+        case 'description':
+          if (updateValue.length > 0) {
+            dispatchUpdateCaseProperty({
+              updateKey: 'description',
+              updateValue,
+            });
+            setIsEditDescription(false);
+          }
+          break;
+        case 'tags':
+          setTags(updateValue as string[]);
+          if (updateValue.length > 0) {
+            dispatchUpdateCaseProperty({
+              updateKey: 'tags',
+              updateValue,
+            });
+            setIsEditTags(false);
+          }
+          break;
+        default:
+          return null;
+      }
+    },
+    [dispatchUpdateCaseProperty, title]
+  );
+
+  useEffect(() => {
+    const caseState = isCaseOpen ? 'open' : 'closed';
+    if (data.state !== caseState) {
+      dispatchUpdateCaseProperty({
+        updateKey: 'state',
+        updateValue: caseState,
+      });
+    }
+  }, [isCaseOpen]);
+  const propertyActions = [
```

Comment:
I know we talk about it, but maybe we can add a `TO DO` item in the file here to clean up propertActions and userActions